### ISSUE TITLE
Spender/ Sammler choice

### ISSUE THREAD

**Issue by tombein** [tombein/capstone-project#2]

**Comment by tombein** [comment]
# Value Proposition

**As a**User
**I need to** choose between Sammler/Spender.
**to be able to** choose between "create offers" and "reserve offers"

# Description
<img width="155" alt="Bildschirmfoto 2022-05-11 um 11 02 24" src="https://user-images.githubusercontent.com/101577162/167814876-75449c03-8d65-45e9-bf42-5ec6731e3f38.png">

# Acceptance Criteria

- [ ] A Button is displayed with label "Sammler"
- [ ] A Button is displayed with label "Spender"


# Tasks

(for developers team)

- [ ] create a new branch "spender-sammler"
- [ ] Create a new component to render the project card in a `<section>` Tag
- [ ] The components includes the two buttons.
- [ ] If the button "sammler" is clicked, the user is navigated to the menu page. 
- [ ] If the button "spender" is clicked, the user is navigated to the add offer. 
- [ ] Apply basic styling (to distinguish between headlines and dynamic values)
- [ ] Add component to the app
- [ ] Add test to check if data is rendered
 
  

# Complexity
S

**Comment by PReichetanz** [comment]
Hello Tom,

the story itself looks good! 
- Both tasks "If the button...is clicked...navigated to" are rather acceptance criteria.
- I think you should add the wireframes of the menu page and add offer (page) where the user is redirected.
- Question: Do these pages already exist as soon as you build this story?
- "Apply basic styling...": There are no headlines and dynamic values in this story, so why would you have to do this now?

Just to be sure: Does "hier meine erste User Story" mean you want to develop this story first (because it has the highest priority?) If so, I would say "no":
- You need to have the menu page and add offer (page) where the user gets redirected; otherwise, the buttons do not do anything when you are finished with this story (and we never want any elements which do not do anything at the end of a story).

Long story short: You can just adjust the tasks / acceptance criteria and then simply postpone this story until you have the menu page / add offer (or whatever).

**Comment by tombein** [comment]
Hello Paul, 

first of all thank you for your feedback. So if I understood you correctly, it makes more sense to start with this card. https://github.com/tombein/capstone-project/issues/3

**Comment by PReichetanz** [comment]
> Hello Paul,
> 
> first of all thank you for your feedback. So if I understood you correctly, it makes more sense to start with this card. #3

If this is the core feature of your app (displaying different kinds of data in Cards), you can create a list of them (with hard coded data for the beginning) in your first story, yes!
